src/components/FindMovie/FindMovie.tsx

```diff
@@ -1,7 +1,67 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './FindMovie.scss';
+import classNames from 'classnames';
+import { Movie } from '../../types/Movie';
+import { getMovie } from '../../api';
+import { MovieCard } from '../MovieCard';
+
+interface P {
+  addMovie: (movie: Movie) => void;
+}
+
+export const FindMovie: React.FC<P> = ({ addMovie }) => {
+  const [query, setQuery] = useState('');
+  const [movie, setMovie] = useState<Movie | null>(null);
+  const [foundMovieError, setFoundMovieError] = useState(false);
+  const [isLoading, setIsLoading] = useState(false);
+
+  const handleAddToList = (event: React.MouseEvent<HTMLButtonElement>) => {
+    if (!movie) {
+      return;
+    }
+
+    event.preventDefault();
+    addMovie(movie);
+    setMovie(null);
+    setQuery('');
+  };
+
+  const handleFindMovie = (event: React.MouseEvent<HTMLButtonElement>) => {
+    event.preventDefault();
```

Comment:
don't send a request if you have an invalid `title`, just with spaces, for example
Also - put an error like `Put the movie name`